cmd/errtrace/main.go

```diff
@@ -607,9 +609,9 @@ func (cmd *mainCmd) readFile(r fileRequest) ([]byte, error) {
 type walker struct {
 	// Inputs
 
-	fset     *token.FileSet // file set for positional information
-	errtrace string         // name of the errtrace package
-	logger   *log.Logger
+	fset        *token.FileSet // file set for positional information
+	errtracePkg string         // name of the errtrace package
```

Comment:
This rename because the command's "unused identifier" logic is quite simple and will consider the name used even if it's a struct field.
We can make it smarter in the future if needed.